common/api-review/dummy-exp.api.md

```diff
@@ -4,18 +4,149 @@
 
 ```ts
 
+import { LogLevel } from '@firebase/logger';
```

Comment:
Remove this file and remove api-report related scripts from dummy-exp package. You don't need it for dummy-exp or size-analysis packages